asciidoc/modules/con_mg_changes-in-eventbus.adoc

```diff
@@ -27,12 +27,15 @@ eventBus.consumer("the-address", message -> {
 });
 ----
 
-The following is an example of request and reply to a message in {VertX} {v4}:
+The following example shows request and reply to a message in {VertX} {v4}.
```

Comment:
_shows a request and a reply_

Reply:
Updated to:
The following example shows the request and reply to a message in {VertX} {v4}.